pkg/operator/k8sutil/labels.go

```diff
@@ -45,3 +46,15 @@ func ParseStringToLabels(in string) map[string]string {
 
 	return labels
 }
+
+// AddRecommendedLabels adds the labels to the resources created by rook
+// The labels added are name, unique_instance,etc
+func AddRecommendedLabels(labels map[string]string, appName, resourceName, resourceKind, resourceInstance string) {
```

Comment:
After some further consideration, I'm not sure the interpretation of `name` as the binary running is truly necessary. Instead, I think we will do better by using the daemon type here (e.g., "mon", "rgw", etc.)

```suggestion
func AddRecommendedLabels(labels map[string]string, daemonType, resourceName, resourceKind, resourceInstance string) {
```